about.html

```diff
@@ -0,0 +1,125 @@
+            <li><a><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a>Logout</a></li>
+            <li><a>Mypage</a></li>
+            <li><a href="https://en.wikipedia.org/wiki/Morocco">Morocco</a></li>
+        </ul>
+    </div>
+    <ul class="mobile-menu">
+        <li><a href="./index.html" class="links" id="homeLink" >Home</a></li>
+        <li><a href="./about.html" class="links" id="aboutLink" >About</a></li>
+        <li><a id="programLink" class="links">Program</a></li>
+        <li><a id="joinLink" class="links">Join</a></li>
+        <li><a id="contactLink" class="links">Contact</a></li>
+    </ul>
+</div>
+<!-- End -->
+
+<main>
+    <section class="headline">
+        <nav class="navbar">
+            <i class="fa-solid fa-bars"  onclick="OpenMenu()" id="menu-icon"></i>
+            <div class="desktop-menu">
+                   <a id="logo" id="homeLink" href="./index.html"> <img src="style/images/logo.png" alt="logo"></a>
+                <ul>
+                     <li><a href="./about.html" id="aboutLink" id="active">About</a></li>
+                    <li><a href="#">Program</a> </li>
+                    <li><a href="#">Join</a> </li>
+                    <li><a href="#">Contact</a> </li>
+                    <li><a href="#" class="compaign">Compaign</a> </li>
+                </ul>
+            </div>
+        </nav>
+    </section>
+
+    <section class="headline2">
+        <div class="tittle">
+            <h3>"Hello! Coding Amateur"</h3>
+            <h1>SUMMER CRASH BOOTCAMP </h1>
+        </div>
+        
+        <div class="headline21">
+            <p class="headline-desc2">Our summer Bootcamp comes with new features, it brings together a community of experts, academics, and activists who comprise great tech corporations. This year, we hope to expand our student list from all over the world.</p>
+            <p class="p">Please contact us per Email for any further questions about summer Bootcamp!</p>
+            <p class="address">summer.bootcamp2023@morocco.org</p>
+        </div>
+
+        <div class="headline-desc21">
+            <h2>Summer BootCamp<br class="null"> Crash Courses 2023</h2>
+            <hr>
+            <p>The course kicks off on June 20 and ends on August 15, 2023.</p>
+            <img class="boot-image" src="style/images/logo.png" alt="">
+        </div>
+        <div class="past-camp">
+            <h3>See the past summer BootCamp</h3>
+            <hr>
+            <p>Take a look at the last two Summer BootCamp whic took place in Geneva and in Cape Town</p>
+
+            <div class="img-all">
+                <div class="image-container1 image-content">
+                        <h3>2022</h3>
+                        <p>Our BootCamp in Geneva</p>
+                </div>
+                <div class="image-container2 image-content">
+                        <h3>2021</h3>
+                        <p>Our BootCamp in Cape Town</p>
+                </div> 
+            </div>
```

Comment:
- [ ] Your images are great but their sizing is not consistent due to the use of `vh` and `vw` values to `height` and `width` attributes, and the right one is smaller than the other. Kindly fix the sizing attributes(I recommend not using `vw` or `vh`) to make them look consistent and with the same size.
 - Your design:
 - 
<img width="1026" alt="image" src="https://user-images.githubusercontent.com/61919617/203775745-a4129d20-d595-4af7-82ef-27a28cbcbe21.png">

 - Template:
 - 
<img width="755" alt="image" src="https://user-images.githubusercontent.com/61919617/203775936-c36e7350-d635-4bef-be64-d60f2a7115e3.png">